ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -636,6 +654,8 @@
             };
         },
         applyConfig: function(config) {
+            // validate siteUrl
+            config.siteUrl = this.validateSiteUrl(config.siteUrl);
```

Comment:
Please, take a look at this [video](https://drive.google.com/open?id=1FGDQcSYe6MlebI0TnrfPpBWHSIOX3-_g)
A user should see the error only in a case if `Submit` button was pressed.